spec/graphql/queries/dogs_spec.rb

```diff
@@ -1,44 +1,118 @@
 require 'rails_helper'
 
-RSpec.describe "dogs query", type: :request do
-  it 'returns all dogs' do
-    user1 = create(:user)
-    dog11 = create(:dog, user: user1)
-    user2 = create(:user)
-    dog21, dog22 = create_list(:dog, 2, user: user2)
-    photo = create(:photo, photoable: dog11)
+RSpec.describe 'dogs query', type: :request do
+  before :each do
+    @u1 = create(:user)
+    @d1 = create(:dog, user: @u1, breed: 'Rat Terrier', activity_level: 0, birthdate: '2015-03-18', weight: 20)
+    @d2 = create(:dog, user: @u1, breed: 'Malinois', activity_level: 1, birthdate: '2010-07-01' , weight: 60)
+    @d3 = create(:dog, user: @u1, breed: 'Tibetan Terrier', activity_level: 2, birthdate: '2002-10-04', weight: 100)
+    @p1 = create(:photo, photoable: @d1)
 
+    @u2 = create(:user)
+    @d4 = create(:dog, user: @u2, breed: 'Rat Terrier', activity_level: 0, birthdate: '2015-03-18', weight: 20)
+    @d5 = create(:dog, user: @u2, breed: 'Malinois', activity_level: 1, birthdate: '2010-07-01', weight: 60)
```

Comment:
```suggestion
    @d5 = create(:dog, user: @u2, breed: 'Malinois', activity_level: 1, birthdate: '2010-07-01', weight: 75)
```
Later on you filter by weight 60-100 -- let's make sure it captures all values in between, not just the edges.

Reply:
Done